spacksites/process-env-scripts/spack-deps-rhel-7.8.sh

```diff
@@ -11,11 +11,11 @@
 #
 # This code was contributed by Owain Kenway
 
-UCL_SCLS="rh-python38 devtoolset-11"
```

Comment:
In `/opt/rh` I only have/see `devtoolset-11`.

Reply:
This is an issue on login12 - it exists everywhere else.

Reply:
Ah, wow, looks like I got the lucky node :slightly_smiling_face:  Fixed with latest push, thanks!